docs/docs/api/LayoutAnimations/AnimatedLayout.md

```diff
@@ -0,0 +1,23 @@
+---
+id: layout_animations_animated_layout
```

Comment:
I think we don't need to prefix ids with namespace since we have to use paths anyway.

We also use camel case in other docs' ids so this id should be "animatedLayout" and filename should start from lowercase.